### PR TITLE
Update relevant files in device-random for Go 1.13.

### LINKED ISSUE #100: Update Minimum Go Version to 1.13
Update references for previous versions of Go to 1.13 in the following files

- Dockerfile
- Dockerfile.build
- go.mod
- Jenkinsfile

### PR DESCRIPTION
**Pull request by akramtexas**
Fix #100

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,6 +37,6 @@ FROM alpine
 ENV APP_PORT=49988
 EXPOSE $APP_PORT
 
-COPY --from=builder /device-random/cmd /
+COPY --from=builder /go/src/github.com/edgexfoundry/device-random/cmd /
```
Comment (akramtexas) [inline Dockerfile:40]: The update to this `COPY` directive was to fix an _existing_ error encountered when running `make docker`, as follows:
```
   ---> 4e7ffef6d1b4
  Step 14/16 : COPY --from=builder /device-random/cmd /
  COPY failed: stat /var/lib/docker/overlay2/6e093e7a8a64ec10212b71126e6b0171391e3527921bbe5b17bb46d409d77665/merged/device-random/cmd: no such file or directory
  make: *** [docker_device_random_go] Error 1
```

**Review by lenny-goodell — COMMENTED** [review]
Your commit message references wrong repo.

**Comment by akramtexas** [comment]
> Your commit message references wrong repo.

Yes, I had caught that and was planning to circle back and amend the Git comment. Will do, thanks 👍

**Comment by akramtexas** [comment]
> Your commit message references wrong repo.

@lenny-intel - Done.  Amended the Git comment.

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Comment by michaelestrin** [comment]
@akramtexas 

image

Compare pull request title with repo name.

**Review by lenny-goodell — APPROVED** [review]

**Comment by lenny-goodell** [comment]
@akramtexas someone for DS WG should review and merge. Added @cloudxxx8 & @iain-anderson

**Pull request merged**